app/models/outcome_with_metadata.rb

```diff
@@ -1,3 +1,7 @@
 class OutcomeWithMetadata < Outcome
   self.table_name = "outcomes_with_metadata"
+
+  def nickname
```

Comment:
Good point. I didn't see `nickname` as a column and forgot that it inherits from `outcome` already.